packages/svelte-cloudinary/tests/GetCldOgImageUrl/GetCldOgImageUrl.test.ts

```diff
@@ -0,0 +1,55 @@
+import { render, screen, cleanup } from '@testing-library/svelte';
+import { afterEach, describe, expect, it, vi } from 'vitest';
+import { CldOgImage, getCldOgImageUrl } from '$src/index';
+import { tick } from 'svelte';
+
+describe('GetCldOgImage', () => {
+	afterEach(() => {
+		cleanup();
+	});
```

Comment:
```suggestion
import { describe, expect, it, vi } from 'vitest';
import { CldOgImage, getCldOgImageUrl } from '$src/index';
import { tick } from 'svelte';

describe('GetCldOgImage', () => {
```

We don't need this since we're not using testing-library in this file!